runhouse/resources/hardware/cluster.py

```diff
@@ -1946,3 +1954,75 @@ def _folder_mv(
 
     def _folder_exists(self, path: Union[str, Path]):
         return self.client.folder_exists(path=path)
+
+    ###############################
+    # Cluster list
+    ###############################
+    @classmethod
+    def list(
+        cls,
+        show_all: Optional[bool] = False,
+        since: Optional[str] = None,
+        status: Optional[ResourceServerStatus] = None,
+    ):
+        """
+        Returns user's runhouse clusters saved in Den. If filters are provided, only clusters that are matching the
```

Comment:
should distinguish between the sky clusters only saved locally and those in Den 

```suggestion
        Returns clusters saved in Den and locally via Sky. If filters are provided, only clusters that are matching the
```